ui/hooks/src/utils/entry-utils.ts

```diff
@@ -124,5 +124,6 @@ export const mapBeamEntryData = (
     createdAt: beam.createdAt,
     content: beam.content,
     nsfw: beam?.nsfw,
+    tags: beam?.tags?.map(labeledTag => labeledTag.value),
```

Comment:
I think this requires first a filter over the `labelType` to be of `sdk.current.services.gql.labelTypes.TAG`

Reply:
Only tags should be there, but I can add the filter as an extra safeguard